tools/misc/tpm2_print.c

```diff
@@ -292,6 +295,42 @@ static bool print_TPM2B_PUBLIC(FILE *fstream) {
     return true;
 }
 
+static bool print_TSSPRIVKEY_OBJ(FILE *fstream) {
+
+    UNUSED(fstream);
+
+    bool retval = true;
+    TSSPRIVKEY_OBJ *tpk = NULL;
+    tool_rc rc = tpm2_util_object_fetch_tpk(ctx.file.path, &tpk);
```

Comment:
Since fetch_tpk and fetch_priv_pub_from_tpk are always called together, can they be a single function?

Reply:
Yes, I think they can be the same function. I added it so that there is a separate function for just getting the tpk struct but was never sure of how useful it was

Reply:
Yes let's combine these into a single call.